drizzlepac/alignimages.py

```diff
@@ -1169,7 +1206,16 @@ def interpret_fit_rms(tweakwcs_output, reference_catalog):
                     'generate_source_catalogs() generate the .reg region files for every chip of every input image and '
                     'should generate_astrometric_catalog() generate file "refcatalog.cat"? Unless explicitly set, the '
                     'default is "False".')
-    'Should the code generate '
+
+    PARSER.add_argument( '-r', '--regtest', required=False,choices=['True','False'],default='False',help='Flag to '
```

Comment:
I **strongly** recommend the advice from https://stackoverflow.com/a/15008806 be followed (also see `'store_true'` and `'store_false'` in https://docs.python.org/3/library/argparse.html#action). This will make the code behave as most people expect, get rid of https://github.com/spacetelescope/drizzlepac/blob/629028ed18dd52649c253236b2efb8f6924e93da/drizzlepac/alignimages.py#L1233-L1248 and of the function in https://github.com/spacetelescope/drizzlepac/blob/629028ed18dd52649c253236b2efb8f6924e93da/drizzlepac/alignimages.py#L160-L172 which, by the way, is unnecessarily complicated: you could simply have:
```python
def convert_string_tf_to_boolean(invalue):
    return invalue == 'True'
```
(which should be done directly where it is needed - no need to define a function for such a simple statement).

Reply:
This code is being pulled out into its own file and being reworked.

Reply:
Issue addressed in PR #298